.github/workflows/linting.yml

```diff
@@ -43,6 +43,9 @@ jobs:
       - name: Check out the repository
         uses: actions/checkout@v2
 
+      - name: Delete generated files
+        run: rm -rf pkg/client pkg/apis/submariner.io/v1/zz_generated.deepcopy.go pkg/natdiscovery/proto/natdiscovery.pb.go
+
```

Comment:
What is this change for?

Reply:
Fair question, it’s to avoid checking generated files; but it’s not great, I’m just going to replace the GHA we use instead.

Reply:
We need to ignore generated files for some part of the license checks; hopefully it’s clearer in the updated version.